src/main/java/org/moltimate/moltimatebackend/service/LigandService.java

```diff
@@ -112,4 +134,67 @@ public String getEcNumber(Structure structure) throws InvalidPdbIdException {
         return EcNumber.UNKNOWN;
     }
 
+    /**
+     * call the Protein Data bank's Search api to get the PDB Ids from the input EC Class
+     * @param ecNumber
+     * @return List of pdbIds in the EC class
+     */
+    public List<String> getPdbIdsFromEcClass(String ecNumber) throws IOException {
+        List<String> pdbIds = new ArrayList<>();
+
+        String USER_AGENT = "Mozilla/5.0";
+
+        String baseURL = "https://search.rcsb.org/rcsbsearch/v1/query?json=";
```

Comment:
I've only partially read Herbert's post; are we certain that this is not the legacy API?

Reply:
Yes we are sure this isn't the legacy API.